concordia/views.py

```diff
@@ -209,25 +211,62 @@ def registration_rate(self, group, request):
 @method_decorator(never_cache, name="dispatch")
 class ConcordiaRegistrationView(RatelimitMixin, RegistrationView):
     form_class = UserRegistrationForm
+    ratelimit_group = "registration"
     ratelimit_key = "ip"
     ratelimit_rate = registration_rate
     ratelimit_method = "POST"
-    ratelimit_block = True
+    ratelimit_block = settings.RATELIMIT_BLOCK
 
 
 @method_decorator(never_cache, name="dispatch")
 class ConcordiaLoginView(RatelimitMixin, LoginView):
+    ratelimit_group = "login"
     ratelimit_key = "ip"
     ratelimit_rate = "3/15m"
     ratelimit_method = "POST"
-    ratelimit_block = True
+    ratelimit_block = False
+
+    def post(self, request, *args, **kwargs):
+        """
+        Handle POST requests: instantiate a form instance with the passed
+        POST variables and then check if it's valid.
+        """
+        form = self.get_form()
+        if form.is_valid():
+            return self.form_valid(form)
+        else:
+            if is_ratelimited(
+                request,
+                group=self.ratelimit_group,
+                key=self.ratelimit_key,
+                method=self.ratelimit_method,
+                rate=self.ratelimit_rate,
+            ):
+                raise Ratelimited(
+                    "We're sorry. Due to too many failed log in attempts "
+                    "we have locked your account. Please try again later "
+                    "or reset your password."
```

Comment:
Does the password reset actually bypass this limit?

Reply:
No it doesn't @elainekamlley supplied that text

Reply:
This is going to be moot since the logic with captcha won't raise any Ratelimited exception.